openupgrade_scripts/scripts/mass_mailing/15.0.2.5/noupdate_changes.xml

```diff
@@ -1,8 +1,6 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
   <record id="ir_cron_mass_mailing_queue" model="ir.cron">
-    <field name="interval_number">1</field>
```

Comment:
Comment lines instead of removing them. That way, we know that is not something forgotten, and we can document why we comment them.